templates/default/luigi-server.init.erb

```diff
@@ -0,0 +1,158 @@
+#! /bin/sh
```

Comment:
Consider creating systemd unit config for systems that moved to systemd (like Ubuntu 16.04).